### PR TITLE
RHMAP-12150 README.md update with doc generation

### PR DESCRIPTION
**Pull request by jcesarmobile**
gh-pages doesn't exist right now in feedhenry repo, we have to [review my branch first](https://github.com/jcesarmobile/fh-ios-swift-sdk/tree/gh-pages) and push the content feedhenry repo.
Once that is done we can review and merge this PR

[Preview of the docs 
](https://jcesarmobile.github.io/fh-ios-swift-sdk/FeedHenry/docset/index.html)

### CODE REVIEW

**Comment by jcesarmobile** [comment]
@danielpassos @aidenkeating can you review?

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ git push origin {TAG}
 
 ### c) Generate API Documentation
 
-To generate API documentation and sync with the [GitHub pages placeholder](http://feedhenry.github.io/fh-ios-sdk/FH/docset/Contents/Resources/Documents/index.html), switch to ['gh-pages'](https://github.com/feedhenry/fh-ios-sdk/tree/gh-pages) branch and follow the instructions there.
+To generate API documentation and sync with the [GitHub pages placeholder](http://feedhenry.github.io/fh-ios-swift-sdk/FeedHenry/docset/index.html), switch to ['gh-pages'](https://github.com/feedhenry/fh-ios-swift-sdk/tree/gh-pages) branch and follow the instructions there.
```
Comment (jcesarmobile) [inline README.md]: It's correct, but first we have to bring the gh-pages branch from my fork 
https://github.com/jcesarmobile/fh-ios-swift-sdk/tree/gh-pages
preview https://jcesarmobile.github.io/fh-ios-swift-sdk/FeedHenry/docset/index.html

Can you review my branch first? then I push it here too and the links should work

**Comment by danielpassos** [comment]
👍 Go ahead

File: README.md (hunk 1 above)
Comment (danielpassos) [inline README.md]: 👍 Go ahead

Comment (jcesarmobile) [inline README.md]: I have pushed the gh-pages branch, are you ok with merging this PR too?

**Pull request merged**